### ISSUE TITLE
WCNFPlus parser does not work

### ISSUE THREAD

**Issue by rjungbeck** [pysathq/pysat#30]
```
import pysat.formula

wcnfp.append([list(range(100)), 50], is_atmost=True)
wcnfp.to_file("test.wcnf")
wcnfp1=pysat.formula.WCNFPlus(from_file="test.wcnf")
```

Produces: 
````
 wcnfp1=pysat.formula.WCNFPlus(from_file="test.wcnf")
  File "formula.py", line 1535, in __init__
    from_string=from_string, comment_lead=comment_lead)
  File "formula.py", line 918, in __init__
    self.from_file(from_file, comment_lead, compressed_with='use_ext')
  File "formula.py", line 958, in from_file
    self.from_fp(fobj.fp, comment_lead)
  File "formula.py", line 1576, in from_fp
    cl = [int(l) for l in line.split()[:-1]]
  File "formula.py", line 1576, in <listcomp>
    cl = [int(l) for l in line.split()[:-1]]
ValueError: invalid literal for int() with base 10: '<='
````

The reason seems to be the wrong test for atmost clause in formula.py 1575 
````
 if line[-1] == '0':  # normal clause
`````

which is true whenever the cardinality value ends with a 0 digit.

**Comment by alexeyignatiev** [comment]
Hi @rjungbeck,

You are right. It should be replaced with `if int(line.rsplit(' ', 1)[-1]) == 0`.

Your example has another problem though. Zero literals are forbidden.

Thanks,
Alexey

**Comment by rjungbeck** [comment]
Lliteral 0 was my unintended fault. (The test case was just produced to show the error)

 Your change seems to work

**Comment by alexeyignatiev** [comment]
This should be fixed in version 0.1.4.dev25. Please, check.
